experiments/benchmarks/utils/config/integrator_config_pwquad.yaml

```diff
@@ -0,0 +1,26 @@
+flow: pwquad
```

Comment:
This line is the only new thing compared to `integrator_config_pwquad.yaml` unless I am mistaken. Since it seems to be the case that pwquad is much better than pwlinear, just edit `integrator_config_pwquad.yaml` to use it and remove this file.